.github/workflows/wheels.yaml

```
@@ -16,7 +16,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ "ubuntu-20.04", "macos-latest" ]
-        python: [ "3.8" , "3.9", "3.10" ]
+        python: [ "3.8" , "3.9", "3.10", "3.11" ]
         platform: [ "x86_64" ]
         macos-target: [ "10.10" ]
```

10.10 is really old. 11 is the oldest supported versions by Apple, but let's take one version older, 10.15, to give a bit more margin.
```suggestion
        macos-target: [ "10.15" ]
```